source/connection-monitoring-and-pooling/tests/pool-clear-min-size.yml

```diff
@@ -0,0 +1,36 @@
+version: 1
+style: unit
+description: pool clear halts background minPoolSize establishments
+poolOptions:
+  minPoolSize: 1
+operations:
+  - name: ready
+  - name: waitForEvent
+    event: ConnectionReady
+    count: 1
+  - name: clear
+  # ensure no connections created after clear
+  - name: wait
+    ms: 200
+  - name: ready
+  - name: waitForEvent
+    event: ConnectionReady
+    count: 1
```

Comment:
This should be `count: 2` otherwise we won't actually wait for the 2nd connection to be created.

Reply:
good catch, fixed.